src/components/victory-area/helper-methods.js

```diff
@@ -3,10 +3,13 @@ import { Helpers } from "victory-core";
 import Data from "../../helpers/data";
 import Domain from "../../helpers/domain";
 import Scale from "../../helpers/scale";
+import Size from "../../helpers/size";
 
 export default {
 
-  getBaseProps(props, defaultStyles) {
+  getBaseProps(props, defaultStyles, defaultWidthHeight) {
+    defaultStyles = props.theme && props.theme.area ? props.theme.area : defaultStyles;
+    props = Object.assign({}, props, Size.getWidthHeight(props, defaultWidthHeight));
```

Comment:
This works, and I see _why_ you're doing it but... I'm really fussy about code style and communication, and I'm having a bit of trouble with this line. I can't think of a better way right now, but let's pair on this on Monday and see what we can come up with.